### PR TITLE
[#3031] New instance bash

### PR DESCRIPTION
**Pull request by tangrammer**
- [ ] **Update release notes if necessary**

### CODE REVIEW

File: ci/new-instance.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+fi
+
+
+log "URL: $URL"
+log "ORGANISATION $ORGANISATION"
+log "EMAIL $EMAIL"
+
+gcloud container clusters get-credentials "$CLUSTER" --zone europe-west1-d --project akvo-lumen
+
+export POD_NAME=$(kubectl get pods -l "run=lumen-blue-green,color=$(./live-color.sh)" -o jsonpath="{.items[0].metadata.name}")
```
Comment (dlebrero) [inline ci/new-instance.sh:46]: export is unnecessary.

File: ci/new-instance.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+log "POD $POD_NAME"
+
+kubectl exec "$POD_NAME" -c lumen-backend -- java -cp akvo-lumen.jar clojure.main -m akvo.lumen.admin.add-tenant "$URL" "$$ORGANISATION" "$EMAIL"
```
Comment (dlebrero) [inline ci/new-instance.sh:50]: $$ORGANIZATION <-- is it $ or $$?

File: ci/new-instance.sh [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+read -r -p "user email:" EMAIL
+
+if [ -z "${EMAIL}" ]; then
+    log "Nothing typed"
+    exit 2
+fi
+
+
+log "URL: $URL"
```
Comment (dlebrero) [inline ci/new-instance.sh:40]: Do you want to validate the url? I see in the doc that there are some requirements. 

Or maybe at least put that requirement in the "read URL" in line 18

File: ci/new-instance.sh (hunk 1 above)
Comment (dlebrero) [inline ci/new-instance.sh:46]: The "./live-color.sh" path means that the expectation is that the script is run from within the `ci` directory. 

Either:
1. Add this to the documentation.
2. Check that the POD_NAME is not empty
3. Avoid the issue as per https://github.com/akvo/akvo-lumen/blob/4b1ac11ac88c8869fa189f8fcd141e1b84843333/ci/flip-blue-green-deployment.sh#L35

**Review by dlebrero — CHANGES_REQUESTED** [review]
The new tenant.md documentation should be updated to point to this file.

File: ci/new-instance.sh (hunk 1 above)
Comment (tangrammer) [reply ci/new-instance.sh]: included

File: ci/new-instance.sh (hunk 3 above)
Comment (tangrammer) [reply ci/new-instance.sh:40]: added requirement in the read URL step

File: ci/new-instance.sh (hunk 2 above)
Comment (tangrammer) [reply ci/new-instance.sh]: fixed

File: ci/new-instance.sh (hunk 1 above)
Comment (tangrammer) [reply ci/new-instance.sh]: removed

**Pull request merged**